core/kits/manager.php

```diff
@@ -119,6 +121,22 @@ public function create( array $kit_data = [], array $kit_meta_data = [] ) {
 		return $kit->get_id();
 	}
 
+	public function create_new_kit( $kit_name = '', $settings = [], $active = true ) {
+		$kit_name = $kit_name ? $kit_name : esc_html__( 'Custom', 'elementor' );
+
+		$id = $this->create( [
+			'post_title' => $kit_name,
+			'settings' => $settings,
+		] );
+
+		if ( $active ) {
+			update_option( self::OPTION_PREVIOUS, $this->get_active_id() );
```

Comment:
Is it possible that $this->get_active_id() won't return ID?

Reply:
No, because the key 'elementor_active_kit'(that contained the id) in wp_options table is generated in the instalation of Elementor plugin.

Reply:
The key 'elementor_active_kit'(that contained the id) in wp_options table is generated in the installation of Elementor plugin and always exists even if the kit has been deleted.

In my opinion we need to delete the 'elementor_active_kit' key when someone delete the key, but this is for another task.